<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>画图</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionADD"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionzoomin_2"/>
    <addaction name="actionzoomout_2"/>
    <addaction name="actionrestore_2"/>
    <addaction name="actionspin"/>
    <addaction name="actionstretch"/>
    <addaction name="actiondelete_all"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actiondrawing_technique"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout_software"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>35</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset theme="画图工具">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;画图工具&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#5500ff;&quot;&gt;画图工具&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="accessibleName">
    <string>画图工具</string>
   </property>
   <property name="styleSheet">
    <string notr="true">画图工具
</string>
   </property>
   <property name="windowTitle">
    <string>画图工具</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QComboBox" name="patternComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>铅笔</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>直线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>矩形</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>椭圆</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>橡皮</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="penStyleComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔样式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔线宽</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="penWidthSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="penColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:black</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="brushColor">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>450</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>填充样式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="brushPatternComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QComboBox" name="colorComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="ColorSelect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>540</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>更多</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="penRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="brushRadioButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>填充</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>选择图形</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionADD">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="statusTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionzoomin_2">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="statusTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionzoomout_2">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="statusTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionrestore_2">
   <property name="text">
    <string>还原</string>
   </property>
   <property name="statusTip">
    <string>还原</string>
   </property>
  </action>
  <action name="actionspin">
   <property name="text">
    <string>旋转</string>
   </property>
   <property name="statusTip">
    <string>旋转</string>
   </property>
  </action>
  <action name="actionstretch">
   <property name="text">
    <string>拉伸</string>
   </property>
   <property name="statusTip">
    <string>拉伸</string>
   </property>
  </action>
  <action name="actiondelete_all">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="statusTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actiondrawing_technique">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>绘图工具</string>
   </property>
   <property name="statusTip">
    <string>绘图工具</string>
   </property>
  </action>
  <action name="actionabout_software">
   <property name="text">
    <string>关于软件</string>
   </property>
   <property name="statusTip">
    <string>关于软件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
